app/src/main/java/org/simple/clinic/summary/PatientSummaryUpdate.kt

```diff
@@ -93,10 +93,34 @@ class PatientSummaryUpdate(
       HasDiabetesClicked -> dispatch(MarkDiabetesDiagnosis(model.patientUuid))
       is HasHypertensionClicked -> hasHypertensionClicked(event.continueToDiabetesDiagnosisWarning, model.patientUuid)
       is HypertensionNotNowClicked -> hypertensionNotNowClicked(event.continueToDiabetesDiagnosisWarning)
-      is StatinPrescriptionCheckInfoLoaded -> noChange()
+      is StatinPrescriptionCheckInfoLoaded -> statinPrescriptionCheckInfoLoaded(event, model)
     }
   }
 
+  private fun statinPrescriptionCheckInfoLoaded(
+      event: StatinPrescriptionCheckInfoLoaded,
+      model: PatientSummaryModel
+  ): Next<PatientSummaryModel, PatientSummaryEffect> {
+    val minAgeForStatin = 40
+    val hasCVD = with(event.medicalHistory) {
+      hasHadStroke == Yes || hasHadHeartAttack == Yes
+    }
+    val isPatientEligibleForStatin = event.age >= minAgeForStatin &&
```

Comment:
This should be OR condition.
Patients eligible for statins:
Patient is diabetic and aged >=40, OR
Patient has history of heart attack or stroke